CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to the "vscode-sops" extension will be documented in this fi
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## [Unreleased]
+### Added
+- Configuration `sops.enabled`, `sops.binPath` works now as expected.
+- Run control (config) file `.vscodesopsrc` for local project/workspace specific configuration (AWS profile & GCP credentials).
+- Support for adjusting `sops.defaultAwsProfile` & `sops.defaultGcpCredentialsPath` configuration of vscode extension.
```

Comment:
no use prefixing these with 'default' I think, as they're a direct setting. Also
`awsProfile` and `gcpCredentialsPath` are shorter :)

Might be worth adding docs that say "empty > system defaults"

Reply:
I'll change that with prefixing "defaults." dot notation. From the clean code point of view, it's more obvious.